### PR TITLE
fix: accounts production url changes

### PR DESCRIPTION
**Pull request by Hardeep8587**
accounts production url changes

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist: (Check only applicable)

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by Hardeep8587** [comment]
@kunalfleethawks please review and merge, thanku

File: src/environments/environment.prod.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ export const environment = {
 
   BaseUrl: 'https://service.cloud.fleethawks.com/api/v1/',
   AssetURL: 'https://fh-cloud-service-uploads-ue1.s3.us-east-1.amazonaws.com',
-  AccountServiceUrl: 'http://localhost:4002/api/v1/',
+  AccountServiceUrl: 'https://accounts.cloud.fleethawks.com/api/v1/',
   SafetyServiceUrl: 'http://localhost:4000/api/v1/',
```
Comment (kunalfleethawks) [inline src/environments/environment.prod.ts]: Please change safety url as well.

Comment (kunalfleethawks) [reply src/environments/environment.prod.ts]: Enable accouting and safety on production

**Review by kunalfleethawks — CHANGES_REQUESTED** [review]
Please address two comments

File: src/environments/environment.prod.ts (hunk 1 above)
Comment (Hardeep8587) [reply src/environments/environment.prod.ts]: Done Kunal

**Comment by Hardeep8587** [comment]
@kunalfleethawks Please check

**Review by kunalfleethawks — APPROVED** [review]

**Pull request merged**